Directory.Build.props

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <!--This should be passed from the VSTS build-->
     <!-- This needs to be greater than or equal to the validation baseline version -->
-    <MicrosoftIdentityWebVersion Condition="'$(MicrosoftIdentityWebVersion)' == ''">3.8.1</MicrosoftIdentityWebVersion>
+    <MicrosoftIdentityWebVersion Condition="'$(MicrosoftIdentityWebVersion)' == ''">3.8.2</MicrosoftIdentityWebVersion>
```

Comment:
What about updating the version of Wilson?

Reply:
It was already updated: https://github.com/AzureAD/microsoft-identity-web/pull/3307/files